packages/node/test/helpers.ts

```diff
@@ -43,9 +44,17 @@ export class TestProcess {
   // a users requests a terminate.
   terminate() {
     if (this.isWin32) {
-      //TODO:
+      // @ts-ignore
+      this.process.stdout.end();
+      // @ts-ignore
+      this.process.stderr.end();
+      try {
+        ExecForTaskkill("taskkill /pid " + this.process.pid + " /T");
+      } catch (e) {
+        // swallow error
```

Comment:
If we need to make terminate an async function so that you can await `ExectForTaskkill` so that you also swallow async errors, that's fine too.